build.csx

```diff
@@ -92,4 +92,38 @@ targets.Add(
     }
 );
 
+targets.Add(
+    "run-gateway-demo",
+    DependsOn("find-msbuild", "restore-demos"),
+    () =>
+    {
+        var salesAPIHost = new Process();
+        salesAPIHost.StartInfo = new ProcessStartInfo
+        {
+            FileName = @".\demos\asp-net-core\Divergent.Sales.API.Host\bin\debug\Divergent.Sales.API.Host.exe",
+            UseShellExecute = true
+        };
+        
+        salesAPIHost.Start();
```

Comment:
@adamralph instead of this verbosity here should I just add a `CmdNoWait` command in the `cmd.csx` script?
Because if we wanna add a target here for every demo and every exercise, the verbosity will explode